tests/Kiota.Builder.Tests/Writers/Php/CodeMethodWriterTests.cs

```diff
@@ -2508,4 +2508,23 @@ public void WritesRequestGeneratorContentTypeQuotes()
         var result = stringWriter.ToString();
         Assert.Contains("\"application/json; profile=\\\"CamelCase\\\"\"", result);
     }
+    [Fact]
+    public void WritesRequestGeneratorBodyForMultipart()
+    {
+        setup();
+        method.Kind = CodeMethodKind.RequestGenerator;
+        method.HttpMethod = HttpMethod.Post;
+        AddRequestProperties();
+        AddRequestBodyParameters();
+        method.Parameters.OfKind(CodeParameterKind.RequestBody)!.Type = new CodeType
+        {
+            Name = "MultipartBody",
+            IsExternal = true
+        };
+        method.RequestBodyContentType = "multipart/form-data";
+        languageWriter.Write(method);
+        var result = stringWriter.ToString();
+        Assert.Contains("MultipartBody $body", result);
```

Comment:
Should this be 
```suggestion
        Assert.Contains("MultiPartBody $body", result);
```

It looks inconsistent with the naming/replacements made in the refiner...

Reply:
It's because we're not doing a refine step in the test case.